lib/symlink.go

```diff
@@ -11,36 +11,53 @@ import (
 )
 
 // CreateSymlink : create symlink or copy file to bin directory if windows
-func CreateSymlink(cwd string, dir string) error {
+func CreateSymlink(target string, link string) error {
 	// If we are on windows the symlink is not working correctly.
 	// Copy the desired terraform binary to the path environment.
 	if runtime.GOOS == windows {
-		r, err := os.Open(cwd)
+		r, err := os.Open(target)
 		if err != nil {
-			return fmt.Errorf("Unable to open source binary: %q", cwd)
+			return fmt.Errorf("Unable to open source binary %q: %v", target, err)
 		}
 		defer r.Close()
 
-		w, err := os.Create(dir)
+		w, err := os.Create(link)
 		if err != nil {
-			return fmt.Errorf("Could not create target binary: %q", dir)
+			return fmt.Errorf("Could not create target binary %q: %v", link, err)
 		}
 		defer func() {
 			if c := w.Close(); err == nil {
 				err = c
 			}
 		}()
+		logger.Infof("Copying binary from %q to %q", target, link)
```

Comment:
I get the "symlinks are a unix/linux thing".. but symlinks are a thing on Windows... I wonder how well it would work if the windows-only branch didn't exist 🤔 
Not that I have access to a (at least modern) windows machine to try :/